src/components/CardTable/CardTable.jsx

```diff
@@ -109,14 +109,16 @@ export default class CardTable extends Component {
   }
 
   get largestTableWidth () {
-    const tableNodes = Object.values(this.tableNodes)
+    const tableNodes = Object.values(this.tableNodes || {})
     const tableWidths = tableNodes.map(tableNode => tableNode.clientWidth)
 
     return Math.max(...tableWidths)
   }
 
   get parentWidth () {
-    return getElementContentWidth(this.containerNode.parentNode)
+    const ZERO = 0
+
+    return this.containerNode ? getElementContentWidth(this.containerNode.parentNode) : ZERO
```

Comment:
Wait no it won't work as it overwrites the containerNode to null. Are you ok with just 

```javascript
get parentWidth () {
  if (this.containerNode === null) {
    const ZERO = 0

    return ZERO
  }

  return getElementContentWidth(this.containerNode.parentNode)
}
```